### PR TITLE
Fixed a typo in openscenario_parser.py

### PR DESCRIPTION
**Pull request by dx6nxr**
#### Description

There was a typo in function "get_traffic_light_from_osc_name" in openscenario_parser.py, which was trying to access "CCarlaDataProvider" insted of "CarlaDataProvider". Typo fixed.

This is the typo.
image
Runs without problems after fixing the typo.

#### Where has this been tested?

  * **Platform(s):** Win11
  * **Python version(s):** 3.8.10
  * **CARLA version:** 0.9.15

#### Possible Drawbacks

None.

- - -
This change is [Reviewable](https://reviewable.io/reviews/carla-simulator/scenario_runner/1118)

### CODE REVIEW

File: result
Comment (Daraan) [inline result:1]: I think you introduced a new file here by accident.

**Review by Daraan — CHANGES_REQUESTED** [review]
I've spotted this a while ago as well (and forgot about it) I wondered more strongly why is this code not executed which would reveal the error, but it seems you have a setup that takes this conditional path.

File: result
Comment (dx6nxr) [reply result:1]: Hey, yes indeed I did. Pushed a new commit deleting this file 👍🏿

**Review by dx6nxr — COMMENTED** [review]
The "results" file is removed.

**Comment by dx6nxr** [comment]
> I've spotted this a while ago as well (and forgot about it) I wondered more strongly why is this code not executed which would reveal the error, but it seems you have a setup that takes this conditional path.

As there are two options for traffic lights, either they have id's in the .xodr file (which is id="" option), and Carla ones, where the traffic light is being obtained by its coordinates (pos=x,y). The test suite probably does not cover the Carla maps and therefore the Varla traffic light representation via the coordinates. For me the traffic lights worked without problems after this change.

Best,
Ivan

**Review by Daraan — APPROVED** [review]

**Comment by Daraan** [comment]
I am sadly no maintainer, but hopefully @PabloVD can merge this typo fix asap.

**Review by PabloVD — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/carla-simulator/scenario_runner/1118)* status: all files reviewed, 1 unresolved discussion (waiting on @Daraan)

**Review by PabloVD — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/carla-simulator/scenario_runner/1118)* status: all files reviewed, 1 unresolved discussion (waiting on @Daraan)

**Review by glopezdiest — APPROVED** [review]
:lgtm:

> Reviewed 1 of 2 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/carla-simulator/scenario_runner/1118#-:-O9igdIpDQLY919C1ctz:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @Daraan)

**Comment by PabloVD** [comment]
Thank you both for spotting this!
@Daraan seems that you are blocking merging the PR with the unresolved discussion, can you resolve it so we can proceed?

**Review by Daraan — COMMENTED** [review]
:lgtm: Took me a bit to realize it is on Reviewable. Hope it is fine now.

> Reviewed 1 of 2 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/carla-simulator/scenario_runner/1118#-:-O9j96VNDlQgHZV_QRuJ:bjr8krg)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @dx6nxr)

**Comment by dx6nxr** [comment]
> :lgtm: Took me a bit to realize it is on Reviewable. Hope it is fine now.
> 
> > Reviewed 1 of 2 files at r1, 1 of 1 files at r2, all commit messages.
> > _[Reviewable](https://reviewable.io/reviews/carla-simulator/scenario_runner/1118#-:-O9j96VNDlQgHZV_QRuJ:bjr8krg)_ status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @dx6nxr)

Didn't know that I have to review it too, sorry, took some time. Should be fine now.

**Comment by Daraan** [comment]
> Didn't know that I have to review it too, sorry, took some time. Should be fine now.

I do not think you have to; Reviewable just automatically tags you in the message it generates.

**Comment by PabloVD** [comment]
Merged, thank you both again!

**Pull request merged**
